README.md

```diff
@@ -142,6 +142,12 @@ You can generate an update site using [Maven](http://maven.apache.org/):
 This will create an update site in
 _com.github.eclipsecolortheme.updatesite/target/repository_.
 
+By default, Eclipse Color Theme will be built against the Eclipse Mars
+version of the platform. You can build against either Eclipse Luna or
+Kepler by setting the "platform-version" property, for example:
+
+    mvn clean verify -Dplatform-version=kepler
+
 License
```

Comment:
I do not suggest distributing more than version of the plugin. I merely suggest that you build/test your plugin against all versions of the platform that you intend to support. (This easily allows you to have three CI jobs to run tests against all three supported platforms.)

The point is that the platform may deprecate and remove APIs over time, but if you only ever build/test against the oldest version of the platform, you will not know that your plugin is broken until it is too late when users report it you.

The version of the platform it builds against by default is kinda irrelevant, I can change it to Kepler by default if that is what you'd prefer.